recipes/mastercard/runner.sh

```diff
@@ -82,4 +101,9 @@ VERSION=$DATE
     Upload $NAME latest
```

Comment:
Is $NAME the same as "output/daily_transactions.zip"?

Reply:
NAME=$(basename $BASEDIR). Name is probably "mastercard"
When one calls Upload, it uploads all files in the output directory. Upload is a user-created function.

Reply:
I see.